docs/mine/lotus/miner-wallets.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Lotus Miner: Wallets'
+description: 'Miners make use of wallets managed by the Lotus Node to both send and receive FIL during the normal course of operation (sending proofs, collecting deal payments etc.). This guides explains how to configure different wallets so the miner can use them for different operations.'
+breadcrumb: 'Miner Wallets'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+As mentioned during the [init phase](miner-setup.md) of the Lotus Miner, it is required to provide an owner address with some funds that the miner will use for all the operations needed (or it will use the [Lotus Node default wallet](../../get-started/lotus/send-and-receive-fil.md)). This address will be used by the miner for all the operations, but in many cases, it makes sense to use dedicated wallets:
+
+However, different components of the miners can be configured to use different wallets:
+
+[[TOC]]
+
+## The owner address
+
+The owner address is used to receive payments and to pay the fees for any miner-actor triggered changes like changing its information on changing addresses (TODO: actually list exactly what it is used for).
+
+You should have specified it during the `lotus-miner init` phase (otherwise, the default Lotus wallet will be used).
+
+This wallet should have enough balance to pay for actor-actions like changing the information.
+
+Given the high value of this wallet, we recommend taking special measures to keep it safe (i.e. keep it in cold storage as long as there are no miner actions to take and the address is not re-used for something else).
+
+## The worker address
+
+The worker address is used to sign blocks and submit proofs associated with this miner. It must be a BLS address and can be specified during `init` with the `--worker` flag.
+
+The worker address is used to sign and send various types of messages (WHICH ONES? including the createStorageMiner message), which means it needs to pay for the gas of those messages. Make sure you keep an eye on its balance!
```

Comment:
What messages exactly? We need a list, particularly if control addresses are defined.

Reply:
Every message [here](https://github.com/filecoin-project/specs-actors/blob/405294d18d84d0d4a0ac35eb156bbdfe81a5e94c/actors/builtin/miner/miner_actor.go#L40-L60) EXCEPT:

- ChangeWorkerAddress (only owner)
- ConfirmUpdateWorkerKey (only owner)
- WithdrawBalance (only owner)
- ConfirmSectorProofsValid (only the Power actor -- cannot be called by anyone)
- OnDeferredCronEvent (only the Power actor -- cannot be called by anyone)
- ApplyRewards (only the Reward actor -- cannot be called by anyone)

Reply:
It also sends messages to the market actor (`PublishStorageDeals` and some balance-related stuff), and to the payment channel actors.